one-ui/src/main/java/com/marklogic/hub/oneui/auth/LoginHandler.java

```diff
@@ -47,6 +47,9 @@ public void onAuthenticationSuccess(HttpServletRequest request, HttpServletRespo
         if (token.getRoles() != null) {
             jsonResponse.putArray("roles").addAll(token.getRoles());
         }
+        if (token.getAuthorites() != null) {
```

Comment:
"Authorites" is misspelled, but if it were spelled correctly, you'd get a compilation error because "getAuthorities" already exists. 

We don't need to store the authorities in two separate fields in AuthenticationToken. Instead, just add a "getAuthorityNames" method that returns a List<String>. It will iterate over each GrantedAuthority and get its value and remove the "ROLE_", since the UI doesn't want to deal with that.

Reply:
oops, i did not notice that. and will modify it.